ui/src/App.tsx

```diff
@@ -105,6 +105,14 @@ export default function App() {
     app.loadGraphData(defaultObj);
   };
 
+  const handleCopyFlow = (json: string, name: string) => {
```

Comment:
can you make handleCopyFlow() a method in WorkspaceContext.ts so you don't need to pass it layers by layers

Reply:
I can't do it cause i change state in App.tsx file.

Reply:
the method is still defined here so you can still access any state you want, you just pass it to <WorkspaceContext.provider value={handleCopyFlow} />